src/Command/Ingester/CreateUserCommand.php

```diff
@@ -0,0 +1,461 @@
+        AssignmentRepository $assignmentRepository,
+        LtiInstanceRepository $ltiInstanceRepository,
+        UserPasswordEncoderInterface $passwordEncoder,
+        NativeUserRepository $userRepository,
+        AssignmentIngester $assignmentIngester,
+        Filesystem $filesystem
+        ) {
+        parent::__construct(self::NAME);
+        $this->lineItemRepository = $lineItemRepository;
+        $this->assignmentRepository = $assignmentRepository;
+        $this->ltiInstanceRepository = $ltiInstanceRepository;
+        $this->passwordEncoder = $passwordEncoder;
+        $this->userRepository = $userRepository;
+        $this->assignmentIngester = $assignmentIngester;
+        $this->filesystem = $filesystem;
+    }
+
+    protected function configure(): void
+    {
+        parent::configure();
+
+        $this->setDescription('Automate a user generate list');
+
+        $this->addOption(
+            self::OPTION_LINE_ITEM_IDS,
+            'i',
+            InputOption::VALUE_REQUIRED,
+            'Comma separated list of line item IDs',
+        );
+
+        $this->addOption(
+            self::OPTION_LINE_ITEM_SLUGS,
+            's',
+            InputOption::VALUE_REQUIRED,
+            'Comma separated list of line item slugs',
+        );
+
+        $this->addOption(
+            'batch',
+            'b',
+            InputOption::VALUE_REQUIRED,
+            'User Create Batch size',
+            self::DEFAULT_BATCH_SIZE
+        );
+
+        $this->addArgument(
+            'user-prefix',
+            InputArgument::REQUIRED,
+            'user prefix list'
+        );
+
+        $this->addOption(
+            self::OPTION_GROUP_PREFIX,
+            'p',
+            InputOption::VALUE_REQUIRED,
+            'Group Prefix',
+        );
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output): void
+    {
+        $this->symfonyStyle = new SymfonyStyle($input, $output);
+        $this->symfonyStyle->title('Simple Roster - Automate the user-generation');
+
+        if (!empty($input->getOption(self::OPTION_LINE_ITEM_IDS)) && !empty($input->getOption(self::OPTION_LINE_ITEM_SLUGS))) {
+            throw new InvalidArgumentException(
+                sprintf(
+                    "Option '%s' and '%s' are exclusive options.",
+                    self::OPTION_LINE_ITEM_IDS,
+                    self::OPTION_LINE_ITEM_SLUGS
+                )
+            );
+        }
+
+        if ($input->getOption(self::OPTION_LINE_ITEM_IDS)) {
+            $this->initializeLineItemIdsOption($input);
+            $criteria = $this->getFindLineItemCriteria();
+            $lineItemsCollection = $this->lineItemRepository->findLineItemsByCriteria($criteria);
+            if(!empty($lineItemsCollection)) {
+                $slugArr = [];
+                foreach($lineItemsCollection as $lineVal){
+                    array_push($slugArr, $lineVal->getSlug());
```

Comment:
$slugArr[] = $lineVal->getSlug() ?

Reply:
Removed above mentioned line as there is a change in the logic of fetching complete line Items collection